load-testing/tests/relays_stress_helpers_test.go

```diff
@@ -517,29 +513,34 @@ func (s *relaysSuite) sendFundAvailableActorsTx(
 	}
 
 	// Fund accounts for **all** suppliers that will be used over the duration of the test.
-	for i := int64(0); i < plans.suppliers.maxActorCount; i++ {
-		// It is assumed that the suppliers keyNames are sequential, starting from 1
-		// and that the keyName is formatted as "supplier%d".
-		keyName := fmt.Sprintf("supplier%d", i+1)
-		supplier := s.addActor(keyName, supplierStakeAmount)
+	suppliersAdded := int64(0)
+	for _, supplierAddress := range s.availableSupplierAddresses {
+		if suppliersAdded > plans.suppliers.maxActorCount {
```

Comment:
Not a blocker, but if we start counting at 0, we reach the max when it's `>=`, not `>`.

Ditto elsewhere.